config/kubernetes/sig-docs/teams.yaml

```diff
@@ -86,6 +86,15 @@ teams:
     - smana
     - yastij
     privacy: closed
+  sig-docs-hi-owners:
+    description: Admins for Hindi content
+    members:
+    - mittalyashu
```

Comment:
I think @daminisatya and @Rajakavitha1 should also be here, since they can help shepherd things.

Reply:
Sure thing. What about you @nikhita? Should I add your name too?

Reply:
> What about you @nikhita? Should I add your name too?

Thanks for asking, but please don't add me. I'm happy to help with reviews and move things along but I don't have too much bandwidth right now. :)

Reply:
Oh! Okay.